test/unit_tests/datasets/test_dataset.py

```diff
@@ -51,8 +51,8 @@ def set_up():
 def concat_ds_targets():
     raws, description = fetch_data_with_moabb(
         dataset_name="BNCI2014001", subject_ids=4)
-    events = mne.find_events(raws[0])
-    targets = events[:, -1]
+    events, _ = mne.events_from_annotations(raws[0])
+    targets = events[:, -1] - 1
```

Comment:
Is the -1 still necessary?

Reply:
The annotations object only stores the string description for events. Thus, the mapping from string to integer is lost. When calling `create_windows_from_events`, I create a custom mapping from 0 on, as we need that for the classification.

`mne.events_from_annotations` internally creates a default mapping when no custom one is specified, that starts from 1. That's why there is a `-1` here.

Another option would be to use a specified mapping as done here:

https://github.com/sbbrandt/braindecode-1/blob/6877169232a5ffcad0ed5073eddb484ec917ad56/braindecode/datautil/windowers.py#L63-L68

Reply:
Got it, thanks!